src/libs/Navigation/AppNavigator/Navigators/CentralPaneNavigator.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import {createStackNavigator} from '@react-navigation/stack';
+
+import SCREENS from '../../../../SCREENS';
+import ReportScreenWrapper from '../ReportScreenWrapper';
+import getCurrentUrl from '../../currentUrl';
+import styles from '../../../../styles/styles';
+
+const Stack = createStackNavigator();
+
+const url = getCurrentUrl();
+const openOnAdminRoom = url ? new URL(url).searchParams.get('openOnAdminRoom') : undefined;
+
+function CentralPaneNavigator() {
+    return (
+        <Stack.Navigator>
+            <Stack.Screen
+                name={SCREENS.REPORT}
+
+                // We do it this way to avoid adding this to url
+                initialParams={{openOnAdminRoom: openOnAdminRoom ? 'true' : undefined}}
```

Comment:
✋ Coming from https://github.com/Expensify/App/issues/28262

This is causing a console error because the type of `openOnAdminRoom` is boolean.